README.md

```diff
@@ -27,6 +27,14 @@ Builders contributed by the public are available in the [Cloud Builders
 Community
 repo](https://github.com/GoogleCloudPlatform/cloud-builders-community).
 
+Each builder includes a `cloudbuild.yaml` that will push your images to [Artifact
+Registry](https://cloud.google.com/artifact-registry) in addition to [Google Container
+Registry](https://cloud.google.com/container-registry). To build with this default `cloudbuild.yaml`,
+you will need to first [create a Docker
+repository](https://cloud.google.com/artifact-registry/docs/docker/store-docker-container-images#create)
+to store the images. The provided `cloudbuild.yaml` assumes your project has set up a [multi-region](https://cloud.google.com/artifact-registry/docs/repositories/repo-locations#location-mr) Artifact
+Registry Docker repositories called `ga` and that is setup for `us`, `europe`, and `asia` multi-regions (i.e. `us-docker.pkg.dev`, `europe-docker.pkg.dev`, `asia-docker.pkg.dev`).
```

Comment:
nit: what's `ga` stand for?

Reply:
Proposal is to have top level dir representing release phases, `ga` stands for `Generally Available`